src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    static ArrayList<Task> list = new ArrayList<>();
+    
+    public static String makeDone(String line) {
+        String msg = "";
+        // sets a specified task as done
+        int index = Integer.valueOf(line.split(" ")[1]);
+        if (index > list.size()) {
+            msg += "No such task!";
+        } else if (list.get(index - 1).isDone) {
+            msg += "Task is already completed!";
+        } else {
+            list.get(index - 1).markAsDone();
+            msg += "Nice! I've marked this task as done: " + '\n';
+            msg += "    " + list.get(index - 1).toString();
+        }
+        return msg;
+    }
+    
+    public static String addTodo(String line) {
+        String msg = "";
+        Task todo = new ToDo(line.substring(5));
```

Comment:
I feel that the magic number here can be better expressed, perhaps declare it as a constant if this magic number will be used extensively.